NOTES.md

```diff
@@ -0,0 +1,35 @@
+# Outline
+
+## Middleware
+
+- Fetches EUR conversion hourly/yearly -- resets cache
+- API:
+  - '/api/currencies/:currencyCode' -- returns currency object with all conversions, adds result to cache
+  - additionally, every return should include a marker indicating how long it will be valid for
```

Comment:
could be a good idea to send back a JWT as well

Reply:
What would be the benefit of a JWT in this scenario?

Reply:
Was trying to think of a way to protect our public endpoint. But it probably doesn't matter in our situation. Short version is we could get an oauth token from Google and they could register it with our app. The more I thought about it though the less necessary it seemed